en/tools/config-catalog-generator/data/configs.json

```diff
@@ -3063,9 +3063,9 @@
             "title": "HTTP/S Pass-through Transport (non-blocking mode)",
             "options": [
                 {
-                    "name": "transport.passthru_http",
```

Comment:
Will it be correct to remove `transport.` part from the config?

Reply:
According to the internal issue it was not working with `transport.passthru_http`. The `passthru_http` config has been tested by the internal team.